lib/test/unit/testcase.rb

```diff
@@ -401,6 +401,14 @@ def description(value, target=nil)
         #     end
         #   end
         #
+        # @example Or in one line (same meaning as above)
```

Comment:
Could you use self-contained description as much as possible?

```suggestion
        # @example Declares that test_do_something_with_ractor uses Ractor in one line
```

Reply:
Yes, thanks for your suggestion!